lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -168,7 +165,7 @@ public void execute(Publisher publisher) throws ConnectorException {
       log.info("Processing rows...");
       while (rs.next()) {
         // Need the ID column from the RS.
-        String id = createDocId(rs.getString(idColumn));
+        String id = idField == null ?  UUID.randomUUID().toString() : createDocId(rs.getString(idColumn));

```

Comment:
Random UUID will make it impossible to reindex into an existing index without creating duplicates. 
So this seems dangerous as a default behavior when idField isn't provided.
Could you tell me more about the situation where you needed to do this?